tests/integration/transformers/test_numerical.py

```diff
@@ -560,3 +561,61 @@ def test_out_of_bounds_reverse_transform(self):
 
         # Assert
         assert isinstance(reverse, pd.DataFrame)
+
+
+class TestLogScaler:
+    def test_learn_rounding(self):
+        """Test that transformer learns rounding scheme from data."""
+        # Setup
+        data = pd.DataFrame({'test': [1.0, np.nan, 1.5]})
+        transformer = LogScaler(
+            missing_value_generation=None,
+            missing_value_replacement='mean',
+            learn_rounding_scheme=True,
+        )
+        expected = pd.DataFrame({'test': [1.0, 1.2, 1.5]})
+
+        # Run
+        transformer.fit(data, 'test')
+        transformed = transformer.transform(data)
+        reversed = transformer.reverse_transform(transformed)
```

Comment:
Can we update this to `reversed_values` or something similar since `reversed` is a Python keyword?